manim/mobject/types/vectorized_mobject.py

```diff
@@ -2056,7 +2059,11 @@ def construct(self):
 
     """
 
-    def __init__(self, *vmobjects, **kwargs):
+    def __init__(
+        self,
+        *vmobjects: VMobject | Iterable[VMobject] | GeneratorType[VMobject],
```

Comment:
```suggestion
        *vmobjects: VMobject | Iterable[VMobject]
```
There's actually no reason to use `GeneratorType[VMobject]`, `GeneratorType` is a subtype of `Iterable`.

Same thing below.

Reply:
Changed this at both places thanks!